relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -108,114 +109,21 @@ pub(crate) fn extract_span_metrics(
                     span_tags.insert(SpanTagKey::Category, category.to_owned());
                 }
 
-                let span_module = if span_op.starts_with("http") {
-                    Some("http")
+                let op_based_tags = if span_op.starts_with("http") {
+                    http_tags::http_span_tags(span)
                 } else if span_op.starts_with("db") {
-                    Some("db")
+                    db_tags::db_span_tags(span)
                 } else if span_op.starts_with("cache") {
-                    Some("cache")
+                    cache_tags::cache_span_tags(span)
                 } else {
-                    None
+                    BTreeMap::new()
                 };
 
-                if let Some(module) = span_module {
-                    span_tags.insert(SpanTagKey::Module, module.to_owned());
-                }
-
-                // TODO(iker): we're relying on the existance of `http.method`
-                // or `db.operation`. This is not guaranteed, and we'll need to
-                // parse the span description in that case.
```

Comment:
The second part of this comment, re `db.operation`, is stale and has been removed.